lib/src/writer.js

```diff
@@ -22,7 +22,8 @@
  */
 
 var imageWrite = require('resin-image-write');
-var umount = require('umount');
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.